app/lib/containers/hyperterm.js

```diff
@@ -42,7 +43,13 @@ class HyperTerm extends Component {
     keys.bind('command+6', moveTo.bind(this, 5));
     keys.bind('command+7', moveTo.bind(this, 6));
     keys.bind('command+8', moveTo.bind(this, 7));
-    keys.bind('command+9', moveTo.bind(this, 8));
+    keys.bind('command+9', () => {
+      const lastTerm = this.terms.getLastTerm();
+      if (lastTerm) {
+        setActiveSession(lastTerm.uid);
```

Comment:
Should this be wrapped in an UI action creator, similar to `moveTo`/`moveLeft` and so on? Maybe to avoid calling setActiveSession if the last one is already active (although this can be checked here as well).